param_persist/serialize/handlers.py

```diff
@@ -0,0 +1,12 @@
+"""
+Handlers used by the deserialize functions to get the correct types.
+
+This file was created on August 04, 2020
+"""
+
+deserialize_handlers = {
+    'int': lambda i: int(i),
```

Comment:
I had envisioned these keys being the associated Param class (e.g. `param.String`). Could also be the string name of the Param class (e.g. "param.String") so that it serializes better. My concern here is down the road with the more complex param objects. What does the type of `param.ObjectSelector` evaluate to? Does that make sense?

Reply:
Couldn't json serialize the class, so i went with the string option.